index.js

```diff
@@ -1,6 +1,37 @@
 const http = require('http');
+const mongoose = require('mongoose');
 require('dotenv').config();
 
+const { Schema } = mongoose;
+
+const movieScheme = new Schema({
+  title: String,
+  overview: String,
+  releaseDate: String,
+  genresIds: [{
+    type: String,
+  }],
+  voteAverage: Number,
+  popularity: Number,
+  originalLanguage: String,
+  voteCount: Number,
+  originalTitle: String,
+  posterPath: String,
+  totalPages: Number,
+});
+
+const Movie = mongoose.model('Movie', movieScheme);
+
+mongoose.connect(`mongodb://localhost/${process.env.APP_PORT}`, { useNewUrlParser: true });
+
+const firstMovie = new Movie({
+  title: 'Fight Clup',
+});
+
+firstMovie.save(() => {
```

Comment:
You trying to save movie, but there is no confidence that MongoDB connection established. First of all you should be sure that db connection established. Try to use mongoose events: `connection.on('error')` and `connection.once('open')`